.changeset/calm-socks-shake.md

```diff
@@ -0,0 +1,6 @@
+---
+"@astrojs/markdown-remark": minor
+"astro": minor
+---
+
+Allow remark plugins to add hProperties to imgs that will be included in the getImage call if they are optimized.
```

Comment:
@OliverSpeir , could you instead describe what the user-facing benefit/feature is here? What does this actually do or unlock? Sounds like it improves using remark plugins? How so? to do what? That's the kind of changeset message that would be helpful to have here.